### PR TITLE
fix duplicate pots in git

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@Rizziepit @smn review please

File: cms/models.py [hunk 1]
```diff
@@ -461,6 +461,12 @@ def auto_save_related_posts_to_git(sender, instance, action, **kwargs):
 
 @receiver(post_save, sender=Post)
 def auto_save_post_to_git(sender, instance, created, **kwargs):
+    # When saving to git, we update the uuid of the instance to ensure
+    # we can find it again.
+    # We need to always retrieve the latest version of the instance to
+    # make sure we always have the uuid if it has been set.
+    instance = Post.objects.get(pk=instance.pk)
```
Comment (Rizziepit) [inline cms/models.py]: Can we rather do this `if not instance.uuid`? This should only ever happen on first save, if I'm not mistaken.

Comment (miltontony) [inline cms/models.py]: We can.

**Comment by Rizziepit** [comment]
@miltontony ready for review. Sorry that I missed this!

**Comment by miltontony** [comment]
cool, I'm merging this

**Pull request merged**